### PR TITLE
Cards container

### PR DESCRIPTION
**Pull request by Giomanconi**

### CODE REVIEW

**Comment by Giomanconi** [comment]
image

File: src/components/Cards.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap/dist/js/bootstrap.bundle.min';
```
Comment (aldoestebanpaz) [inline src/components/Cards.js]: Please import the the bootstrap assets only in the index.js file as explained in the following tutorials:

https://create-react-app.dev/docs/adding-bootstrap/
https://blog.logrocket.com/how-to-use-bootstrap-with-react-a354715d1121/

File: src/components/Cards.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+
+export default function Cards(props) {
+    return (
+
```
Comment (aldoestebanpaz) [inline src/components/Cards.js]: remove unnecessary, without context, empty lines in the middle of the file

File: src/components/CardsContainer.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap/dist/js/bootstrap.bundle.min';
```
Comment (aldoestebanpaz) [inline src/components/CardsContainer.js]: same as Cards.js

File: src/App.css [hunk 4]
```diff
@@ -1,3 +1,17 @@
 .image {
   height: 50px;
 }
+
+.products-cards {
+  width: 250px!important;
+}
+.shop-btn {
```
Comment (aldoestebanpaz) [inline src/App.css]: separate style definitions with a empty line

**Comment by aldoestebanpaz** [comment]
please try to merge the latest change of upstream in your repo in order to fix the conflict

**Pull request merged**